climate_emotions_map/utility.py

```diff
@@ -25,7 +25,6 @@
 
 def get_state_options():
     """Get the options for the state dropdown."""
-    # TODO: The state values include cluster labels in parentheses, e.g. (Cluster F). Can remove from label if desired.
```

Comment:
This would be an app-wide change, right? Since the state dropdown shows the cluster labels too

Reply:
Oh just realized that this is not a change in `make_map.py` 😅 but my question still holds I think -- if the "(Cluster X)" is stripped then it needs to be stripped from the map hoverboxes too

Reply:
It would, but actually Eric already confirmed we don’t need to change this, so we’re good!!